2025-01-20 22:08:59,673 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:09:00,353 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:09:00,354 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:09:00,361 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:09:00,372 INFO: 127.0.0.1 - - [20/Jan/2025 22:09:00] "GET /socket.io/?EIO=4&transport=polling&t=PI5Z1wO HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:09:00,388 INFO: 127.0.0.1 - - [20/Jan/2025 22:09:00] "POST /socket.io/?EIO=4&transport=polling&t=PI5Z1zP&sid=k5gKexb9BScJRof3AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:09:00,390 INFO: 127.0.0.1 - - [20/Jan/2025 22:09:00] "GET /socket.io/?EIO=4&transport=polling&t=PI5Z1zT&sid=k5gKexb9BScJRof3AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:09:27,702 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
